src/main/java/com/juniorstart/juniorstart/model/Goal.java

```diff
@@ -0,0 +1,84 @@
+package com.juniorstart.juniorstart.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+
+import javax.persistence.*;
+import java.sql.Date;
+import java.util.UUID;
+
+@Builder
+@Getter
+@Setter
+@Entity
+@Table(name = "goal")
```

Comment:
zbędne, bo i tak będzie taka nazwa tabeli

Reply:
Tez uważam , że jest to niepotrzebne.